ansible/files/logrotate-postgres

```diff
@@ -1,10 +1,9 @@
-/var/lib/postgresql/12/main/pg_log/postgresql.csv {
-    daily
-    size 50M 
+/var/lib/postgresql/data/pg_log/postgresql.log {
```

Comment:
@dragarcia fyi it'll be broken on the pg13 ami w/o this

Reply:
sure thing, let me rebuild and update the images